pydoctor/epydoc2stan.py

```diff
@@ -785,10 +785,10 @@ def format_summary(obj: model.Documentable) -> Tag:
     source, parsed_doc = _get_parsed_summary(obj)
     if not source:
         source = obj
-
-    # Disallow same_page_optimization in order to make sure we're not
-    # breaking links when including the summaries on other pages.
-    with source.docstring_linker.disable_same_page_optimazation():
+    
+    # do not optimize url in order to make sure we're alwasy geberaing full urls.
```

Comment:
I want to make a suggestion here but I literally don't know what "geberaing" is supposed to be 😅